tests/e2e/ctl_v3_elect_test.go

```diff
@@ -98,7 +100,7 @@ func testElect(cx ctlCtx) {
 }
 
 // ctlV3Elect creates a elect process with a channel listening for when it wins the election.
-func ctlV3Elect(cx ctlCtx, name, proposal string) (*expect.ExpectProcess, <-chan string, error) {
+func ctlV3Elect(cx ctlCtx, name, proposal string, expectFailure bool) (*expect.ExpectProcess, <-chan string, error) {
```

Comment:
I would prefer to handle failure by caller.

Reply:
me too, do you have an idea how I can do this from the go routine down there? Otherwise I'd have to add another error channel

Reply:
Seems ok for now.